SQL/Archive/17.1/2017-05-11-unique-sub-tests.sql

```diff
@@ -1,7 +1,25 @@
 WARNINGS;
 SET SQL_NOTES=1;
 
-SELECT 'Alter instrument_subtests to force unique Subtest_name' as 'Step #1';
+SELECT 'Delete duplicate entries' as 'Step #1';
+DELETE 
```

Comment:
ERROR 1055 (42000): Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'LORIS_17_1.instrument_subtests.ID' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by

I would suggest usign this synthax:
`
DELETE FROM instrument_subtests USING instrument_subtests, instrument_subtests to_keep
  WHERE instrument_subtests.ID < to_keep.ID AND instrument_subtests.Test_name = to_keep.Test_name AND instrument_subtests.Subtest_name = to_keep.Subtest_name;
`
This should keep the latest record according to auto_increment.